$NetBSD: patch-configure,v 1.3 2015/06/29 11:12:45 wiz Exp $

Use USE_AMD64_IOPL (actually x86_64_iopl) on NetBSD/amd64.
(c.f. patch-hw_xfree86_os-support_bsd_i386.c)

--- configure.orig	2015-02-10 22:49:52.000000000 +0000
+++ configure
@@ -22680,7 +22680,7 @@ $as_echo "#define USE_DEV_IO 1" >>confde
 $as_echo "#define USE_DEV_IO 1" >>confdefs.h
  ;;
 		*netbsd*)
-$as_echo "#define USE_I386_IOPL 1" >>confdefs.h
+$as_echo "#define USE_AMD64_IOPL 1" >>confdefs.h
 
 				SYS_LIBS=-lx86_64
 				;;
